### ISSUE TITLE
Several scripts in projects need to quote paths

### ISSUE THREAD

**Issue by wibblymat** [GoogleChrome/sw-testing-helpers#34]
The project scripts (publish-release, etc.) are broken if your current path includes a space.

You need to put quotes around them.

e.g. 

```bash
$SCRIPTPATH/publish-docs.sh releases/$PACKAGE_VERSION
```

should be

```bash
"$SCRIPTPATH/publish-docs.sh" releases/$PACKAGE_VERSION
```

**Comment by gauntface** [comment]
Argh.

Thank you.

**Comment by gauntface** [comment]
This should be fixed 0.0.15
